pkg/lang/ir/custom.go

```diff
@@ -43,7 +44,7 @@ func (g Graph) compileCustomPyPIPackages(root llb.State) llb.State {
 	// Compose the package install command.
 	var sb strings.Builder
 	// Always use the conda's pip.
-	sb.WriteString("pip install")
+	sb.WriteString(`pip install`)
```

Comment:
Does this one require the absolute path?

Reply:
We should not use an absolute path here since users may use conda/venv to install a new python environment.